src/components/charts/galacticProperties/index.jsx

```diff
@@ -289,6 +289,8 @@ class GalacticProperties extends React.Component {
       options,
     } = this.props;
 
+    const { svgShape, svgShapes } = options || {};
```

Comment:
do we need both options?  Or can we just use `svgShapes` and in cases where you're only wanting one always use `svgShapes[0]`?

Reply:
I tried applying this and it didn't work for me. Doesn't quite make sense to me.

Reply:
okie doke